css/properties/-moz-image-region.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "selectors": {
```

Comment:
This should be a property, based on the MDN article https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-image-region